crates/valence_protocol/src/packet/s2c/play/scoreboard_display.rs

```diff
@@ -2,6 +2,44 @@ use crate::{Decode, Encode};
 
 #[derive(Copy, Clone, Debug, Encode, Decode)]
 pub struct ScoreboardDisplayS2c<'a> {
-    pub position: u8,
+    pub position: Position,
     pub score_name: &'a str,
 }
+
+#[derive(Copy, Clone, PartialEq, Debug)]
+pub enum Position {
+    List,
+    Sidebar,
+    BelowName,
+    Team(u8),
```

Comment:
I would prefer to not leave this as a byte if there is a better abstraction. There should also be a short doc comment to explain it.